web/pdf_page_view.js

```diff
@@ -284,6 +288,24 @@ class PDFPageView {
     }
   }
 
+  #renderTextLayer() {
+    // This is a low priority task, so handle the other stuff we've in queue
+    // before this one.
+    setTimeout(() => {
```

Comment:
Why do we actually need to do this, since the textLayer already waits for the page to finish rendering?
By delaying it even more than before, other things (such as e.g. the annotationLayer) will now parse/render first which could *potentially* delay the textLayer noticeably (compared to before). Consider e.g. a pending search operation, during document loading (just to take one easy example).

Reply:
At some point, it helped to remove some sluggishness, but it seems that it's now useless, hence I'll remove that stuff.